.github/workflows/federated-integ-test.yml

```diff
@@ -60,14 +60,14 @@ jobs:
       - name: Build FEG docker images
         run: |
           cd feg/gateway/docker
-          docker-compose build --force-rm --parallel
+          docker compose build --force-rm --parallel
           docker images
       - name: Export docker images to deploy them
         run: |
           mkdir images
           cd images
-          docker save feg_gateway_go:latest  | gzip > fed_feg_gateway_go.tar.gz
-          docker save feg_gateway_python:latest  | gzip > fed_feg_gateway_python.tar.gz
+          docker save feg-gateway_go:latest  | gzip > fed_feg-gateway_go.tar.gz
```

Comment:
Here you change the tarred files to use a dash rather than an underscore, but for the orc8r components above you don't. Is there a reason for this?

Reply:
Let's do the naming of images on the LF artifactory once the switch is done. This is also the reason for the failing test mentioned by @sebathomas.
I'm not sure if we can leave the externally visible package names as they are (with an underscore), or if everything has to be changed to a dash naming scheme.

Reply:
I decided to leave the naming scheme on the artifactory as is. We would need two PRs for this anyway and it would only add up to the complexity of the PR.